package.json

```diff
@@ -82,7 +82,7 @@
     "@github/combobox-nav": "^2.1.5",
     "@github/markdown-toolbar-element": "^2.1.0",
     "@github/paste-markdown": "^1.4.0",
-    "@primer/behaviors": "1.3.0",
+    "@primer/behaviors": "1.3.1-rc.8de3bf8",
```

Comment:
Just published 1.3.1 for @primer/behaviors: https://github.com/primer/behaviors/releases/tag/v1.3.1

Reply:
Thanks @colebemis!